bin/configs/scala-pekko.yaml

```diff
@@ -0,0 +1,6 @@
+generatorName: scala-pekko
+outputDir: samples/client/petstore/scala-pekko
```

Comment:
can you please add the new folder path to .github/workflows/samples-scala.yaml so that the CI will test it moving forward when there's a change to the files in the folder?

Reply:
folder added